Monika After Story/game/script-topics.rpy

```diff
@@ -15694,6 +15696,134 @@ label monika_eating_meat:
+    m 3eua "Which is probably where I would fit in, for example."
+    m 1eud "I told you about how I was feeling kind of in-between, while still being a little more extroverted, remember?"
+    m 1rud "Speaking of...{w=0.4} When thinking about all this, I realised that while this is an arguably pretty important part of one's personality..."
+    m 3rksdla "...I don't actually know where you would fit on that spectrum."
+    m 3eka "And of course, I'm rather curious about knowing this much about you, so..."
+    m 1eua "How would you describe yourself, [player]?"
+    $ _history_list.pop()
+    menu:
+        "How would you describe yourself, [player]?"
+
+        "I'm introverted.":
+            $ persistent._mas_pm_introvert = True
+            $ persistent._mas_pm_extrovert = False
+            $ persistent._mas_pm_ambivert = False
+            $ persistent._mas_pm_social_personality_unsure = False
+            m 1esa "I see."
+            m 4eub "I take it that you usually prefer spending time without too many people over going out within large groups and such?"
+            m 4eka "Don't worry, I can totally understand.{w=0.3} And to tell you the truth..."
+            show monika 5fubla at t11 zorder MAS_MONIKA_Z with dissolve_monika
+            m 5fubla "...the idea of spending my evenings staying at home, cuddling with you does sound lovely."
+            m 5ruc "You know, a lot of people seem to think being an introvert means that you must be a shy person. {w=0.3}{nw}"
+            extend 5wud "But that couldn't be further from the truth!"
+            show monika 4euc at t11 zorder MAS_MONIKA_Z with dissolve_monika
+            m 4euc "A lot of introverts have no trouble at all interacting with other people."
+            m 4eka "They just prefer not to do so too often, and would rather save their energy for activities that they actually enjoy."
+            m 2etc "And besides,{w=0.3} shy extroverts exist too, which only further prove my point, if you ask me."
+            m 2ekbla "In any case, I hope you don't mind sharing your life with someone else at least a little bit, even if it's just me."
+            m 1hublb "I'll try my best to make sure you're always feeling comfortable with me, I promise~"
+
+        "I'm extroverted.":
+            $ persistent._mas_pm_extrovert = True
+            $ persistent._mas_pm_introvert = False
+            $ persistent._mas_pm_ambivert = False
+            $ persistent._mas_pm_social_personality_unsure = False
+            m 3eub "Alright!"
+            m 3eua "So, I suppose you like to spend time with other people, going outside to have fun, and so on?"
+            show monika 5eub at t11 zorder MAS_MONIKA_Z with dissolve_monika
+            m 5eub "I can definitely see the appeal in that.{w=0.3}{nw}"
+            extend 5hub "I'd love to go to a party with you or something, one of these days."
```

Comment:
This line as well, I feel like this also makes an assumption, I assume not every extrovert is into partying and such.